### ISSUE TITLE
Not working after fresh installation

### ISSUE THREAD

**Issue by Cantawe** [homalg-project/NConvex#14]
I did a fresh install of GAP-4.13.1 for Windows using the .exe installer provided at GAP's official
website. After loading the package NormalizInterface and then NConvex, some of NConvex commands
do not work. For instance, I tried with the example from the documentation:

gap> SolveEqualitiesAndInequalitiesOverIntergers( [ [ 3, 5 ] ], [ 8 ], [ [ 4, -2 ], [ 3, 1 ] ], [ 2, 3 ] );
Error, VAL_GVAR: No value bound to NmzCone in
  ValueGlobal( "NmzCone" ) at /opt/gap-4.13.1/pkg/nconvex/gap/Polyhedron.gi:166 called from
ExternalNmzPolyhedron( p ) at /opt/gap-4.13.1/pkg/nconvex/gap/Polyhedron.gi:329 called from
LatticePointsGenerators( P ) at /opt/gap-4.13.1/pkg/nconvex/gap/ZSolve.gi:44 called from
<function "SolveEqualitiesAndInequalitiesOverIntergers">( <arguments> )
 called from read-eval loop at *stdin*:4
type 'quit;' to quit to outer loop

Since ExternalNmzPolyhedron seemed problematic I also tried:

gap> P:=PolyhedronByInequalities([[2,3,4],[1,2,3]]);
<A polyhedron in |R^2>
gap> ExternalNmzPolyhedron(P);
Error, VAL_GVAR: No value bound to NmzCone in
  ValueGlobal( "NmzCone" ) at /opt/gap-4.13.1/pkg/nconvex/gap/Polyhedron.gi:166 called from
<function "ExternalNmzPolyhedron method">( <arguments> )
 called from read-eval loop at *stdin*:5
type 'quit;' to quit to outer loop

Is this a known issue? Thanks.

**Comment by kamalsaleh** [comment]
The error-message `Error, VAL_GVAR: No value bound to NmzCone in` suggests that `NormalizInterface` is not properly installed and hence its methods are not available. I would recommend consulting the installation instructions of `NormalizInterface` or maybe install the gap system in a linux-subsystem (e.g., Ubuntu) in windows to avoid unexpected errors, e.g., https://learn.microsoft.com/en-us/windows/wsl/install.

**Comment by fingolfin** [comment]
Just saw this by chance. Can't comment on the GAP .exe installer myself, but perhaps @ChrisJefferson knows whether NormalizInterface is expected to work in that? I notice that we don't install `flint` in https://github.com/gap-system/gap-windows/blob/main/cygwin-gap-runtime.list so that may be an impediment ... but it also seems cygwin only has an outdated FLINT 2.8 package anyway, and Normaliz these days wants 3.0.1 or later... Hrm

**Comment by ChrisJefferson** [comment]
Works fine in 4.14, on windows. I think almost everything (except float and xgap) work in 4.14.

```
gap> SolveEqualitiesAndInequalitiesOverIntergers( [ [ 3, 5 ] ], [ 8 ], [ [ 4, -2 ], [ 3, 1 ] ], [ 2, 3 ] );
[ [ [ 1, 1 ] ], [ [ 5, -3 ] ], [  ] ]
gap>  P:=PolyhedronByInequalities([[2,3,4],[1,2,3]]);
<A polyhedron in |R^2>
gap> ExternalNmzPolyhedron(P);
<a Normaliz cone>
```

**Comment by mohamed-barakat** [comment]
Could this issue be closed?

**Comment by kamalsaleh** [comment]
Yes, since it was caused by an improper installation of `NormalizInterface`.
